src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +15,11 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
+
     public User findByEmail(String email) {
-        return null;
+        return Arrays.stream(users)
```

Comment:
stream works slower than for each, so for this particular task you can use for each loop